src/components/header.js

```diff
@@ -0,0 +1,44 @@
+import {Navbar,Nav,NavDropdown,FormControl,Button,Form,InputGroup} from 'react-bootstrap';
+import '../App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import * as Icon from 'react-bootstrap-icons';
+
+// Header component will be shown on sign in & Sign up page. It is similar to navbar except that this compenent does not have
+// signup & Signin button as the user is already on one of these two pages.
+function Header(){
+  return (
```

Comment:
NO need no design another Header that appears during sign-in.You should code for that which part appears before sig-in or which part after.